src/SafeMigrationsBundle.php

```diff
@@ -60,7 +41,7 @@ public function loadExtension(array $configs, ContainerConfigurator $containerCo
         $excludedServiceStatements = $this->getExcludedStatements($containerBuilder, $excludedStatements);
 
         $containerBuilder->setDefinition('eniams.safe_migrations.warning_factory', new Definition(WarningFactory::class))
-            ->setArguments([tagged_iterator('eniams.safe_migrations.statement', exclude: $excludedServiceStatements),
+            ->setArguments([tagged_iterator('eniams.safe_migrations.statement'),
```

Comment:
Todo: exclude not available in Symfony 5.4 so extra work needed here

Reply:
A quick win is to pass the excluded statements as a parameter in the `WarningFactory`, then create an array of statements as string and check in the support method where we check if we have to add a warning

```php

public function __construct(private readonly iterable $statements, private readonly iterable $excludedStatements, private readonly array $criticalTables = []){}

private function createWarningBasedOnStatements(string $migration): Warning
    {
        $commandOutputWarning = $migrationWarning = '';
        foreach ($this->statements as $statement) {
            if ($statement->supports($migration) && !in_array($statement->getStatement(), $this->getExcludedStatementAsString()) {
                $commandOutputWarning .= $this->warningFormatter->commandOutputWarning($statement->migrationWarning());
                $migrationWarning .= $this->warningFormatter->migrationWarningLine($statement->migrationWarning());
            }
        }

        return new Warning($commandOutputWarning, $migrationWarning);
    }

private function getExcludedStatementAsString()
{
foreach($this->excludedStatements as $excluded) {
$excludedStatementsAsString[] = $excluded->getStatement();

return $excludedStatementsAsString;
}